cupy/reduction.py

```diff
@@ -414,28 +412,30 @@ def __call__(self, *args, **kwargs):
         out_args = _get_out_args_with_params(
             out_args, out_types, out_shape, self.out_params)
 
+        block_size = 512
         in_indexer = carray.Indexer(in_shape)
         out_indexer = carray.Indexer(out_shape)
-        out_clp2_size = 2 ** int.bit_length(int(out_indexer.size - 1))
+        clp2_count = 2 ** int.bit_length(
```

Comment:
Please write the meaning of this line or extract a function.

`clp2_count` is the minimum power of 2 that satisfies `clp2_count  >= in_indexer.size / out_indexer.size`.